src/main/java/Ejercicio4/Ejercicio2.java

```diff
@@ -0,0 +1,18 @@
+package Ejercicio4;
+
+import java.util.Arrays;
+
+public class Ejercicio2 {
+    public static void main(String[] args) {
+        final var arreglo = Utiles.crearArregloAleatorio(500,10000,100000);
+        System.out.printf("El arreglo divido entre 100 es: %s%n", Arrays.toString(dividirValor(arreglo)));
+
+    }
+    private static double[] dividirValor(double[] arreglo) {
+        final var nuevoArreglo = new double[500];
```

Comment:
no asumas que el tamaño es 500
declara final var n = arraglo.length y lo usas aquí y en el for de abajo